make/test.mk

```diff
@@ -345,3 +345,21 @@ CLEAN_TARGETS += clean-coverage-unit
 # Removes unit test coverage file
 clean-coverage-unit:
 	-@rm -f $(COV_PATH_UNIT)
+
+#-------------------------------------------------------------------------------
+# e2e test in dev mode
+#-------------------------------------------------------------------------------
+
+.PHONY: build-image-local
+build-image-local: e2e-setup
+	eval $$(minishift docker-env) && operator-sdk build $(shell minishift openshift registry)/devconsole-e2e-test/devopsconsole-operator
+
+.PHONY: e2e-local
+e2e-local: build-image-local
+	@-oc project devconsole-e2e-test
+	@-oc apply -f deploy/crds/devopsconsole_v1alpha1_component_crd.yaml
```

Comment:
@baijum fixed here: 7292817b029570c12b651c255982dbd7e851c25d

Reply:
@baijum we don't need to abandon relative paths as we're ALWAYS building inside the source directory. That is common for Go projects and I don't see when this will ever change.